console/console-init/ui/src/modules/iot-device/DeviceListPage.tsx

```diff
@@ -23,13 +24,15 @@ import {
 } from "modules/iot-device/components";
 import {
   getHeaderForDialog,
-  getDetailForDialog
+  getDetailForDialog,
+  MAX_ITEM_TO_DISPLAY_IN_DEVICE_LIST as MAX_DEVICES
```

Comment:
Try to use the const variable as it is or rename the variable

Reply:
Done